yamcs-api/src/main/proto/yamcs/protobuf/filetransfer/filetransfer.proto

```diff
@@ -208,11 +232,8 @@ message CreateTransferRequest {
   // **Required** The path at the remote entity. Example: ``a/local/path/some_filename``.
   optional string remotePath = 5;
   
-  optional DownloadOptions downloadOptions = 6;
+  optional UploadOptions uploadOptions = 6;
```

Comment:
Can you also revert uploadOptions back to its original serialization number 7?
(same reason as before -- i hadn't noticed it before)